packages/astro/src/runtime/astro.ts

```diff
@@ -1,5 +1,7 @@
 import type { AstroComponent, AstroComponentFactory } from '../internal';
 
+import { spreadAttributes, defineStyleVars, defineScriptVars } from '../internal';
```

Comment:
This introduces a circular dependency between `internal/index.js` and `runtime/astro.js`. Since this file was created as a hack to get around Vite trying to prepare SSR modules for the client anyway, I’ll just move this entire file into `internal/index.ts` which was meant for SSR.

Reply:
Would this explain the following result on `npm run dev` from a fresh install of this version?

```bash
> astro-site@0.21.0 dev
> astro dev

Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/Users/thewrays/astro-site/node_modules/astro/dist/internal' is not supported resolving ES modules imported from /Users/thewrays/astro-site/node_modules/astro/dist/runtime/astro.js
    at finalizeResolution (internal/modules/esm/resolve.js:272:17)
    at moduleResolve (internal/modules/esm/resolve.js:699:10)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:810:11)
    at Loader.resolve (internal/modules/esm/loader.js:86:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:230:28)
    at ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:56:40)
    at link (internal/modules/esm/module_job.js:55:36) {
  code: 'ERR_UNSUPPORTED_DIR_IMPORT',
  url: 'file:///Users/thewrays/astro-site/node_modules/astro/dist/internal'
}
```